src/site/stages/build/drupal/load-saved-flags.js

```diff
@@ -12,7 +12,7 @@ const fs = require('fs-extra');
  * @return {Proxy} - A Proxy containing all the feature flags; throws
  *                   an error if a flag is called without existing
  */
-function useFlags(rawFlags, shouldLog = true) {
+function useFlags(rawFlags, shouldLog = false) {
```

Comment:
The logging from this function creates a ton of noise, which makes debugging difficult.

Reply:
Doesn't this only log for the error? This seems important to log if feature flags can't be found.

Reply:
I assumed it was not important since the errors have been spewing 100's of times into the console during the builds for a long time and nobody indicated any concern about them.

The noise from these error messages was interfering with my ability to debug.

If you like, I can undo this change, and spend some time investigating the errors, but this is unplanned work and unrelated to my team's goals.

Reply:
Since it seems unrelated to the fix, I'd lean towards undoing this change. It's up to you whether you want to investigate further.

Reply:
It may be a matter of pulling the latest content from Drupal. I saw many errors when I did a `yarn build:content` on a stale cache, and I didn't see them anymore when I passed `--pull-drupal`.

Reply:
I'll undo the change and see if I have the same experience. Maybe we can reduce the noise and add/or a more helpful error message?